### PR TITLE
Make it impossible to join as non-mapped jobs

### PR DESCRIPTION
**Pull request by soryy708**
Affected jobs: Roboticist, Geneticist, Virologist, Curator, Lawyer, Counselor, Detective

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl:
config: Now impossible to join as: Roboticist, Geneticist, Virologist, Curator, Lawyer, Counselor, Detective
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

### CODE REVIEW

File: config/jobs.txt [hunk 1]
```diff
@@ -22,24 +22,24 @@ Janitor=2,1
 
 Clown=1,1
 Mime=1,1
-Curator=1,1
-Lawyer=2,2
+Curator=0,0
+Lawyer=0,0
```
Comment (Kmc2000) [inline config/jobs.txt:26]: Need to mirror this to the host

**Review by Kmc2000 — DISMISSED** [review]
Nice

File: config/jobs.txt (hunk 1 above)
Comment (soryy708) [reply config/jobs.txt:26]: What?

Comment (Kmc2000) [reply config/jobs.txt:26]: it's a note for me don't worry

**Review by Kmc2000 — APPROVED** [review]

**Pull request merged**